pkg/storage/storage.go

```diff
@@ -147,6 +147,11 @@ func List(client *occlient.Client, componentName string, applicationName string)
 			continue
 		}
 
+		// We should ignore ConfigMap (while PR2142 and PR2601 are not fixed)
+		if client.IsVolumeAnConfigMap(volumeMount.Name, dc) {
```

Comment:
The client currently has no such function. Thus the build is failing
```
pkg/storage/storage.go:151:12: client.IsVolumeAnConfigMap undefined (type *occlient.Client has no field or method IsVolumeAnConfigMap)
```

Reply:
Yes sorry! Just committed it